source/MaterialXCore/Util.cpp

```diff
@@ -101,6 +101,60 @@ string joinStrings(const StringVec& stringVec, const string& sep)
     return res;
 }
 
+StringVec splitListInitializer(const string& str)
```

Comment:
From the text description of this changelist, I see that it's focused on the custom struct syntax for value strings in MaterialX, but this isn't yet reflected in the name of the new function or its documentation.

Should we consider giving this function a name along the lines of `parseStructValueString`?  And would it make more sense as a helper method in `Value.h` and `Value.cpp`, rather than as a core string manipulation method in `Util.h`?

Reply:
I put the function in the string manipulation code because it's only operating on the strings, it felt like a very close sibling to `splitString`.  In fact at one point I even considered making it a "mode" of `splitString` but the code felt clearer once it was separated.

I'm open to moving it to `value.cpp`, but I would worry that in the future developers would think it only relates to struct types, where as the actual function has no ties to structs at all, it just happens to be that we have defined in the specification that the structs are initialized with strings that look like C++ initializer lists.  It feels more discoverable to me if it lives with the other string manipulation code.

Reply:
I could be wrong, but the syntax rules in this helper function seem very closely tied to our custom struct syntax, especially in the use of the semicolon to separate struct members.  I wouldn't expect a developer find this function useful in parsing true C++ initializer lists, for example, and I can't think of other uses in the MaterialX context beyond parsing custom structs.

Maybe I'm missing other examples, though, where this same function would be useful to a MaterialX developer?